ci_test/integration_tests/test_integration_atom_wae.py

```diff
@@ -25,15 +25,16 @@
 num_decoder_layers = 3
 
 # Reconstruction loss
-expected_train_recon_range = (0.500, 0.525)
+#expected_train_recon_range = (0.500, 0.525)
```

Comment:
```suggestion
```
If they aren't used, just get rid of them? If these are left without a comment, nobody will remember why they're there and commented out.